src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import reportWebVitals from './reportWebVitals';
-import App from './App';
+import TodoContainer from './components/TodoContainer';
 import './App.css';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <TodoContainer />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```

Comment:
Since the new version of React, the console is going to give you an error if you use ReactDom like this. Please use create root to avoid this error. You can find more about it here: https://reactjs.org/docs/react-dom-client.html#createroot

Reply:
- Thanks @sebalp100  🙏 for reviewing my pull request I appreciate your review  I will modify👌 this to enhance my code thanks for your hints to make me a professional I am very happy to hear this optional suggestion I will work on it 💯 
### Thanks again for reviewing my PR  and have a good day😊